app/queries/servers/drafts.ts

```diff
@@ -30,3 +29,27 @@ export const queryDraft = (database: Database, channelId: string, rootId = '') =
 export function observeFirstDraft(v: DraftModel[]) {
     return v[0]?.observe() || of$(undefined);
 }
+
+export const queryAllDrafts = (database: Database, teamId: string) => {
```

Comment:
Well is not all drafts is it?

Reply:
Yup, its correct, it for a specific team, will change the name.,